hawkular-javaagent-wildfly-dist/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.hawkular.agent</groupId>
+    <artifactId>hawkular-wildfly-agent-parent</artifactId>
+    <version>0.29.3.Final-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hawkular-javaagent-wildfly-dist</artifactId>
+  <packaging>pom</packaging>
+
+  <name>Hawkular Javaagent: WildFly + Javaagent</name>
+  <description>A WildFly distribution that includes an agent installed inside of it.</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.hawkular.agent</groupId>
+      <artifactId>hawkular-javaagent-wildfly-feature-pack</artifactId>
+      <version>${project.version}</version>
+      <type>zip</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+
+      <!-- provision a plain WildFly server -->
+      <plugin>
+        <groupId>org.wildfly.build</groupId>
+        <artifactId>wildfly-server-provisioning-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>provision-plain-wildfly</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>build</goal>
+            </goals>
+            <configuration>
+              <config-file>server-provisioning-plain-wildfly.xml</config-file>
+              <server-name>${project.build.finalName}</server-name>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- Install the Hawkular WildFly Agent inside the WildFly server -->
+      <!--
+           The agent will be installed in standalone.xml file as well as host.xml and domain.xml.
```

Comment:
I assume this comment is not right. The agent doesn't go in standalone.xml, host.xml, or domain.xml.